src/main/java/com/odysseusinc/arachne/portal/repository/BaseUserRepository.java

```diff
@@ -110,7 +110,7 @@
 
     U findByUuid(String uuid);
 
-    @Query("SELECT u FROM User u")
+    @Query("SELECT u FROM User u WHERE u.enabled=true")
     List<U> getAll();
```

Comment:
Not sure, but seems like it can break user list in admins panel. It should display all users, disregarding their enabled state, and it uses the same method

Reply:
No, method for admin panel uses Pageable. This is only for reindex solr. I can rename it for more readability.